_posts/fr/newsletters/2025-05-30-newsletter.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Bulletin Hebdomadaire Bitcoin Optech #356'
+permalink: /fr/newsletters/2025/05/30/
+name: 2025-05-30-newsletter-fr
+slug: 2025-05-30-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+La newsletter de cette semaine résume une discussion sur les effets possibles des échecs
+attribuables sur la confidentialité du LN. Sont également inclus nos sections régulières avec des
+questions et réponses sélectionnées du Bitcoin Stack Exchange, des annonces de nouvelles versions et
+candidates à la sortie, ainsi que des descriptions des changements récents dans les logiciels
+d'infrastructure Bitcoin populaires.
+
+## Nouvelles
+
+- **Les échecs attribuables réduisent-ils la confidentialité du LN ?** Carla Kirk-Cohen a
+  [posté][kirkcohen af] sur Delving Bitcoin une analyse des conséquences possibles pour la
+  confidentialité des dépensiers et des receveurs du LN si le réseau adopte les [échecs
+  attribuables][topic attributable failures], en particulier en informant le dépensier du temps qu'il
+  a fallu pour transférer un paiement à chaque saut. Citant plusieurs articles, elle décrit deux types
+  d'attaques de désanonymisation :
+
+  - Un attaquant exploitant un ou plusieurs nœuds de transfert utilise les données de temps pour
+    déterminer le nombre de sauts utilisés par un paiement ([HTLC][topic htlc]), ce qui peut être
+    combiné avec la connaissance de la topographie du réseau public pour réduire l'ensemble des nœuds
+    qui auraient pu être le receveur.
+
+  - Un attaquant utilise un achemineur de trafic réseau IP ([système autonome][]) pour surveiller
+    passivement le trafic et combine cela avec la connaissance de la latence du réseau IP entre les
```

Comment:
```suggestion
  - Un attaquant utilise un achemineur de trafic réseau IP
    ([système autonome][]) pour surveiller passivement le trafic et combine cela avec la connaissance de la latence du réseau IP entre les
```

having a link on this line will cause the bullet to be formatted differently than the EN page as well as the other bullets in this section. this newline (hack?) will keep the formatting the same